api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -185,4 +186,14 @@ message HttpProtocolOptions {
 
   // Defines http specific outlier detection parameters.
   OutlierDetection outlier_detection = 8;
+
+  // Specifies a list of HTTP-level mirroring policies to be evaluated for requests
+  // routed to this cluster. Cluster-level mirroring is evaluated after route-level
```

Comment:
Just thinking out loud about this choice for policy behavior. The mirroring policies are "most specific wins". I wonder it makes sense to declare that cluster level policies are "more specific" than route level and apply the same logic. I can see someone wanting to be able to completely override route policy by the cluster policy. With this approach it is not possible. Just wanted to get your point of view on this one.